pkg/cmd/auth/refresh/refresh.go

```diff
@@ -156,14 +157,15 @@ func refreshRun(opts *RefreshOptions) error {
 		return cmdutil.SilentError
 	}
 
-	var additionalScopes []string
+	additionalScopes := set.NewStringSet()
```

Comment:
`set.StringSet` makes sense to use here instead of handing all the adding and removing ourselves.

Reply:
Heh, I was going to come back myself and rewrite this using some kind of set, I didn't know we already had one.

Much better thanks.